code/backend/src/main/java/server/rest/controllers/Controller.java

```diff
@@ -0,0 +1,10 @@
+package server.rest.controllers;
+
+/**
+ * Created by vaast on 30/10/2017.
+ */
+public class Controller {
```

Comment:
I moved these to a config file with a bean defined (however haven't pushed). I'll change this myself once I pull your changes into my local branch.

Reply:
Another side-note, having it in config allows us to define it on the elastic beanstalk environment, so say we were running a different DB in that environment then (local vs hosted) then we wouldn't have any code changes necessary for the connection string to change.